install-and-configure/install/multi-cluster/federated-etl/thanos-migration-guide.md

```diff
@@ -82,21 +90,19 @@ kubectl create secret generic federated-store --from-file=federated-store.yaml -
 
 ### Step 5: Enable FederatedETL, ETL-Utils, and Aggregator on the primary cluster
 
-{% hint style="warning" %}
-*Do not* disable Thanos during this step. Thanos will be disabled in a later step.
-{% endhint %}
 
-Enabling FederatedETL will begin pushing your primary cluster's ETL data to the directory `/federated/CLUSTER_ID`. This setting will remain on once disabling Thanos at a later step.
+Enabling FederatedETL will begin pushing your primary cluster's ETL data to the directory `/federated/CLUSTER_ID`. This setting will be enabled when upgrading to v2.x.
 
 Enabling ETL-Utils will create the directories `/federated/CLUSTER_ID` for every primary/secondary cluster based on the full set of data in the `/etl` directory.
 
-Enabling [Aggregator](/install-and-configure/install/multi-cluster/federated-etl/aggregator.md) will begin processing the ETL data from the `/federated` directory. The aggregator will then serve all Kubecost queries.
+Enabling [Aggregator](/install-and-configure/install/multi-cluster/federated-etl/aggregator.md) will begin processing the ETL data from the `/federated` directory. The aggregator will then serve all Kubecost queries. Be sure to look at the Aggregator Optimizations if Kubecost is ingesting data for ~20,000+ containers. It is difficult to recommend the amount of resources needed due to the uniqueness of each environment and several other variables.
+
+The `federatedStorageConfigSecret`, `etlBucketConfigSecret`, and `thanosSourceBucketSecret` *MUST* all point to the same bucket. Otherwise the data migration will not suceed. Additionally, the cloud-integration *MUST* be configured and referenced following this [method](https://docs.kubecost.com/install-and-configure/install/cloud-integration/multi-cloud#step-2-create-cloud-integration-secret) or the cloud integration will fail.
```

Comment:
```suggestion
The `federatedStorageConfigSecret`, `etlBucketConfigSecret`, and `thanosSourceBucketSecret` *MUST* all point to the same bucket. Otherwise the data migration will not suceed. Additionally, the cloud-integration *MUST* be configured and referenced following [this method](/install-and-configure/install/cloud-integration/multi-cloud#step-2-create-cloud-integration-secret) or the cloud integration will fail.
```